pkg/process/checks/process.go

```diff
@@ -483,6 +494,11 @@ func fmtProcesses(
 			ProcessContext:         serviceExtractor.GetServiceContext(fp.Pid),
 		}
 
+		if tags, ok := pidToGPUTags[fp.Pid]; ok {
+			log.Info("Detected GPU, and process is in activePids, adding gpu tags to pid:", fp.Pid)
```

Comment:
Let's set this to debug logging, also use the formatted one so we actually format and print the pid.